### DISCUSSION TITLE
Can Not mask the webview

### DISCUSSION THREAD

**Discussion by wenyaoshao** [AvaloniaUI/Avalonia#16640] [Q&A]
I want to implement a loading mask, globally. The main page consists of a header and a dockpanel. The DockPanel has a navigation bar on the left and a webview on the right. By design, I should mask the entire dockpanel. 
Now if the webview is not loading, the loading mask works fine. But if the web view is loaded, it appears at the top, blocking part of the loading mask.
So how can I mask the webview on the shell?
![image](https://github.com/user-attachments/assets/c889cd85-d75f-4deb-8297-1aa9f6a44d90)

**Comment by rabbitism** [comment]
What webview are you using? If its implemented with NativeControlHost, then this is by design.
